src/EditorFeatures/CSharpTest/InitializeParameter/InitializeMemberFromParameterTests.cs

```diff
@@ -1517,5 +1517,29 @@ public C(string? s)
     public string? S { get; }
 }", parameters: new TestParameters(options: options.PropertyNamesArePascalCase));
         }
+
+        [WorkItem(24526, "https://github.com/dotnet/roslyn/issues/24526")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInitializeParameter)]
+        public async Task TestSingleLineBlock()
```

Comment:
📝 Also needs a test for code style where braces are _not_ allowed to be on the same line.

Reply:
done.